slick-testkit/src/main/resources/application.conf

```diff
@@ -0,0 +1,7 @@
+h2mem1 = {
```

Comment:
Why are this file and the `slick-testkit/src/test/resources/application.conf` needed instead of using a configuration from `common-test-resources/application.conf`?